src/pytest_inmanta_lsm/remote_orchestrator.py

```diff
@@ -245,18 +246,22 @@ def sync_project(self) -> None:
         if server_version >= Version("5.dev"):
             venv_path: str = os.path.join(server_path, ".env")
             # venv might not exist yet so can't just access its `inmanta` executable -> install via Python script instead
-            install_script_inline: str = (
-                # use only double quotes in script so it can be wrapped in single quotes
+            python_script_inline: str = (
                 "from inmanta.module import Project;"
-                f'project = Project("{server_path}", venv_path="{venv_path}");'
+                f"project = Project('{server_path}', venv_path='{venv_path}');"
                 "project.install_modules();"
             )
+            shell_script_inline: str = (
+                # use the server's environment variables for the installation
+                "sudo systemd-run User=inmanta EnvironmentFile=/etc/sysconfig/inmanta-server"
+                " /opt/inmanta/bin/python -c %s" % shlex.quote(python_script_inline)
```

Comment:
I don't think this is going to work:

* The user should be provided to `systemd-run` via the `--uid` option and not via `User=inmanta`
* An environment file cannot be passed via `EnvironmentFile=/etc/sysconfig/inmanta-server`. I am not even sure whether these is support to pass an environment file to `systemd-run`
* `systemd-run` returns immediately and not when the command returns. The `--wait` option should be used to make this a blocking call.

Reply:
I meant to set the options via `-p` (forgot to add it here). I've tested this on 192.168.2.98 (`systemd-run` with `User` and `EnvironmentFile` supplied, not the entire install). Do you believe it should work like that? I was not familiar with `systemd-run` myself so it's possible I misunderstand how to use it, but from the man pages and my experimenting I'd expect this to work.
I'll add the `--wait` option, good catch.

Reply:
Indeed. With `-p` it should work fine.